draft-ietf-acme-acme.md

```diff
@@ -370,7 +378,7 @@ ACME clients SHOULD send an Accept-Language header field in accordance with
 
 ACME servers that are intended to be generally accessible need to use
 Cross-Origin Resource Sharing (CORS) in order to be accessible from
-browser-based clients {{?W3C.CR-cors-20130129}}.  Such servers SHOULD set the
+browser-based clients {{?W3C.CR-cors-20140116}}.  Such servers SHOULD set the
```

Comment:
This is failing in CI:
> *** 404 Not Found while fetching https://xml2rfc.tools.ietf.org/public/rfc/bibxml4/reference.W3C.CR-cors-20140116.xml

Suggestions from RFC wonks welcome.

Reply:
Looks like [this BibXML exists](https://xml2rfc.tools.ietf.org/public/rfc/bibxml4/reference.W3C.REC-cors-20140116.xml) so `{{?W3C.REC-cors-20140116}}` should work.

Reply:
@bifurcation Thank you! Incorporated in 4d83965 - looks like it did the trick :tada: